Rhea/AppDelegate.m

```diff
@@ -169,6 +169,11 @@ - (void)userNotificationCenter:(NSUserNotificationCenter *)center didActivateNot
     }
 }
 
+- (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center
+     shouldPresentNotification:(NSUserNotification *)notification {
```

Comment:
I didn’t know either! :)

Sure, changing it right now, sorry.


> On 21 Aug 2016, at 19:35, Tim Johnsen <notifications@github.com> wrote:
> 
> In Rhea/AppDelegate.m:
> 
> > @@ -169,6 +169,11 @@ - (void)userNotificationCenter:(NSUserNotificationCenter *)center didActivateNot
> >      }
> >  }
> >  
> > +- (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center
> > +     shouldPresentNotification:(NSUserNotification *)notification {
> 
> Minor nit, but could you use the code formatting I have been using throughout the rest of the project? Would have the method signature all on one line and the{ on the following line, like
> 
> - (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center shouldPresentNotification:(NSUserNotification *)notification
> {
> ...
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>